### ISSUE TITLE
OMEMO: "self" contact "has new devices" with my own key

### ISSUE THREAD

**Issue by RalfJung** [dino/dino#775]
After upgrading to the latest Dino and restarting a few times, Dino now tells me that in the conversation with myself (quite useful to send text between my devices), I have "new devices". When I click on "Manage", it shows a "new key" which is exactly the same as my "Own key".

Looks like Dino is confusing itself into thinking its own key is someone else's?

**Comment by licaon-kter** [comment]
This is still there after a restart?

**Comment by RalfJung** [comment]
Yes, and in fact I get desktop notifications about this on every restart.

**Comment by RalfJung** [comment]
FWIW, this is still a problem with latest master (45c0ac410433089cf973854c146b6b22df4a1a1c).

**Comment by RalfJung** [comment]
After clicking the checkmark to "accept" my own key, the notification is gone.

But it seems a bit silly that I have to tell Dino to accept the key that it generated itself...
